pom.xml

```diff
@@ -26,48 +32,24 @@
 	<developers>
 		<developer>
 			<name>Edward Robinson</name>
```

Comment:
A: Should this refer back to Mr. Robinson's current contact info, since (to my knowledge) he isn't affiliated with Tech Brewery?

Reply:
I think he is - he just hasn't yet gotten to updated Governance requests